editor/editor_file_system.h

```diff
@@ -290,6 +301,8 @@ class EditorFileSystem : public Node {
 	bool _scan_extensions();
 	bool _scan_import_support(Vector<String> reimports);
 
+	bool _try_emit_filesystem_changed();
```

Comment:
Why does this return `bool` if you never use the value?

Reply:
Most try methods I've seen follow the pattern of returning whether the attempt was successful or not. I ended up implementing that here.

Reply:
If the value isn't used anywhere, it should be `void`. If you are that worried about conventions, you could just rename it.

Reply:
Alright, I'll just make it void return type.